### PR TITLE
V0.19.3 merge

### PR DESCRIPTION
**Pull request by asmacdo**
Steps: 

1. Create new local branch `v0.19.3-merge` from `downstream/master`
1. `git fetch upstream` && `git merge v0.19.3`
    a. Solve conflicts. In this case it was just some go.mod lines.
1. Run `go mod vendor`
1. This PR

### CODE REVIEW

File: internal/scaffold/ansible/dockerfilehybrid.go [hunk 1]
```diff
@@ -65,7 +65,8 @@ ENV OPERATOR=/usr/local/bin/ansible-operator \
 RUN yum clean all && rm -rf /var/cache/yum/* \
  && yum -y update \
  && yum install -y libffi-devel openssl-devel python36-devel gcc python3-pip python3-setuptools \
- && pip3 install --no-cache-dir --ignore-installed ipaddress \
+ && pip3 install --no-cache-dir \
```
Comment (asmacdo) [inline internal/scaffold/ansible/dockerfilehybrid.go:68]: Looks like this change didn't make it into https://github.com/openshift/ocp-release-operator-sdk/blob/751d7ef02cd948c4c3591d73e68d7aee9e296c24/release/ansible/upstream.Dockerfile#L28.

@fabianvf, do you make these changes by hand or is there a script? https://github.com/openshift/ocp-release-operator-sdk/commit/f3da6c0518566a59fa8674f07a03a6d1b9195faa

**Comment by asmacdo** [comment]
@fabianvf do we run [`update-collections.sh`](https://github.com/openshift/ocp-release-operator-sdk/blob/751d7ef02cd948c4c3591d73e68d7aee9e296c24/release/ansible/update-collections.sh) on z-stream releases, or (my guess) would that just be for X any Y stream?

File: internal/scaffold/ansible/dockerfilehybrid.go (hunk 1 above)
Comment (fabianvf) [reply internal/scaffold/ansible/dockerfilehybrid.go:68]: a combination of the two, I usually run `cd release/ansible && go run ../../hack/image/ansible/scaffold-ansible-image.go` (same thing for helm, but with the helm scaffolder instead). Might make some untracked files you can ignore, including a Dockerfile which you can then compare to the Dockerfiles in release/ansible.

**Comment by fabianvf** [comment]
@asmacdo I think we can probably skip updating the collections, tbh I don't know if we've done a z-stream release since collections were a thing

**Review by jmrodri — APPROVED** [review]
they look okay.

**Pull request merged**